packages/date/src/TimezonePickerInput.tsx

```diff
@@ -1,32 +1,77 @@
-import React, { ChangeEvent } from 'react';
-import { zoneOffsets, defaultZoneOffset } from './utils/zoneOffsets';
+import React, { useCallback, useEffect, useState } from 'react';
 
-import { Select } from '@contentful/f36-components';
+import { Autocomplete } from '@contentful/f36-components';
+
+import allTimezones, { Timezone } from './utils/timezones';
+import { defaultZoneOffset } from './utils/zoneOffsets';
+
+import dayjs from 'dayjs';
+
+import utc from 'dayjs/plugin/utc';
+import timezone from 'dayjs/plugin/timezone';
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
 
 export type TimezonepickerProps = {
   disabled: boolean;
   onChange: (value: string) => void;
   value?: string;
 };
+
 export const TimezonepickerInput = ({
   disabled,
   onChange,
   value = defaultZoneOffset,
 }: TimezonepickerProps) => {
+  const localTimezone = dayjs.tz.guess();
+  const defaultTimezone =
+    value && value !== dayjs.tz(undefined, localTimezone).format('Z')
```

Comment:
I don't think you need `value && value !==  string` since `value` uses `defaultZoneOffset` by default, and the only way to override this is by passing `null ... <TimezonepickerInput value={null}/>` which will convert the expression to `null && null !== string` which means that we don't need the first part `value &&`